lib/tasks/dev_configuration.rake

```diff
@@ -22,15 +22,16 @@ class RakeDevConfiguration
     end
 
     def toggle_spring
-      file = "tmp/spring-off.txt"
+      file = "tmp/spring-on.txt"
       FileUtils.mkdir_p("tmp")
 
       if File.exist?(file)
         delete_cache_file(file)
-        puts "Spring is now enabled."
+        puts "Spring is no longer enabled (you will need to manually kill the spring processes)"
```

Comment:
might be useful to go ahead and do that here with a system call

Reply:
That seems scary to me, but if it would be useful to you, I'll add it in!

Reply:
Nah, I'm ok waiting to see if the need arises :)